app/src/androidTest/java/net/skyscanner/backpack/docs/DocsRegistry.kt

```diff
@@ -50,11 +50,12 @@ object DocsRegistry {
     ViewScreenshot("Calendar 2 - Pre-selected range", "range", ::setupCalendar2),
     ViewScreenshot("Calendar 2 - Day colours", "colored", ::setupCalendar2),
     ViewScreenshot("Calendar 2 - Day labels", "labeled", ::setupCalendar2),
-    ViewScreenshot("Card - Default", "default"),
-    ViewScreenshot("Card - Without padding", "without-padding"),
-    ViewScreenshot("Card - Selected", "selected"),
-    ViewScreenshot("Card - Corner style large", "corner-style-large"),
-    ViewScreenshot("Card - With divider", "with-divider"),
+    ViewScreenshot("Card - View- Default", "default"),
+    ViewScreenshot("Card - View- Without padding", "without-padding"),
+    ViewScreenshot("Card - View- Selected", "selected"),
+    ViewScreenshot("Card - View- Corner style large", "corner-style-large"),
+    ViewScreenshot("Card - View- With divider", "with-divider"),
+    ComposeScreenshot("Card - Compose", "default"),
```

Comment:
looks like the screenshot wasn't generated. does all the content fit without scrolling?